README.md

```diff
@@ -536,7 +536,10 @@ Hive UDFs are registered with a database. There are currently two limitations in
 Due to the distributed nature of Waggle Dance using UDFs is not that simple. If you would like a UDF to be used from a federated metastore we'd recommend registering the code implementing it in a distributed file or object store that is accessible from any client (for example you could store the UDF's jar file on S3). See creating permanent functions in the [Hive documentation](https://cwiki.apache.org/confluence/display/Hive/LanguageManual+DDL#LanguageManualDDL-Create/Drop/ReloadFunction).
 
 ### Hive metastore filter hook
-The database calls `getDatabases` and `getAllDatabases`, as well as `getTableMeta` do not support having the provided filter applied at the moment, so their result will not be modified by the filter.
+You can configure a Hive filter hook via: `hive-metastore-filter-hook: filter.hook.class`
+This class needs to be on the classpath and can be an external jar. If so the command to run wd needs to be updated to ensure correct class loading this can be done by adding: -Dloader.path=<path_to_jar>
```

Comment:
```suggestion
This class needs to be on the classpath and can be an external jar. If so the command to run needs to be updated to ensure correct class loading. This can be done by adding: `-Dloader.path=<path_to_jar>`
```